### PR TITLE
Product listing

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Preview: https://deploy-preview-2--trekkart-stores.netlify.app/
Added Product Listing Page, added filters
image
image

### CODE REVIEW

File: src/backend/db/products.js [hunk 1]
```diff
@@ -8,23 +8,282 @@ import { v4 as uuid } from "uuid";
...
+    name: "Combo Offer(Bags and Shoes)",
+    description: "Stylish Bags at Best Deals! Grab it!",
+    alt:"bags",
+    newPrice: 2899,
+    originalPrice: 3999,
+    isBestSeller: true,
+    rating: 4.6,
+    brand: "gucci",
+    category: "travel bags",
+		inStock: false,
```
Comment (prakashsakari) [inline src/backend/db/products.js:77]: take care of this indentation

File: src/component/ProductCard/ProductCard.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./ProductCard.css"
+
+export function ProductCard({ product }) {
+  const {
+    _id: id,
+    alt,
+    img,
+    // description,
```
Comment (prakashsakari) [inline src/component/ProductCard/ProductCard.js:10]: if this is your final push then remove the comments

**Review by prakashsakari — APPROVED** [review]
1. Good folder structure
2. Hover effect on cards is amazing
3. Theme and the UI go hand in hand

**Review by jeetbhalani15 — COMMENTED** [review]
-Good UI
-Folders looks good
-clean code

**Pull request merged**